src/_base/harness/scripts/enable.sh.twig

```diff
@@ -77,7 +77,7 @@ dynamic_console()
 {
     if console_enabled; then
         # ensure it is started
-        if [ -z "$("${COMPOSE_BIN[@]}" ps --quiet console)" ]; then
+        if [ -z "$("${COMPOSE_BIN[@]}" ps --quiet --all console)" ]; then
```

Comment:
erm, not actually relevant here, but as it's only reproducing same behaviour as before, there's no new bug